### DISCUSSION TITLE
Accessing Opened Editor Properties

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10422] [Q&A] [answered]
Hi,
Is there any way to access currently opened editor widget and access its path ? Does theia provide with any event regarding that ?

Thanks

**Comment by msujew** [comment, marked as answer]
You could inject the `EditorManager`, which contains a `currentEditor` property, which probably contains what you're looking for. It returns `EditorWidget | undefined`. Assuming any editor is selected, you can use `getResourceUri()` on the `EditorWidget`. That should retrieve the URI of the opened editor.

**Reply by maheshrajrp** [reply]
Hi @msujew ,
Thanks for the reply.  This works. Also, I hope to use `onActiveEditorChanged` event from `EditorManager`.

Thanks !!
